.evergreen/.install_node

```diff
@@ -30,4 +30,15 @@ fi
 
 . "$BASEDIR/.setup_env"
 
-npm --unsafe-perm=true run bootstrap-ci -- --scope @mongosh/build --scope @mongosh/cli-repl --include-dependencies
+# We need the build package for various tasks, and can bootstrap the cli-repl
+# package on all hosts, including dependencies.
+# mongodb-client-encryption cannot be installed everywhere without prerequisites
+# (because it's hard to build from source when there's no prebuilts available
+# because it requires libmongocrypt to be installed globally then), but we still
+# need its types; so we first try to install it, and if that fails, we fall back
+# to installing with --ignore-scripts (i.e. do not attempt to build addons)
+# and only do the TypeScript compilation step, which is sufficient for the
+# executable compilation step.
+npm --unsafe-perm=true run bootstrap-ci -- --scope @mongosh/build
+npm --unsafe-perm=true run bootstrap-ci -- --scope @mongosh/cli-repl --include-dependencies || \
+  (npm --unsafe-perm=true run bootstrap-ci -- --scope @mongosh/cli-repl --include-dependencies --ignore-scripts && npm --unsafe-perm=true run compile-cli)
```

Comment:
For context: This was working with mongodb-client-encryption@1.2.3, because that included a `deps` folder with the necessary headers and library files to build a broken version of the addon (the library files were only actually working on x64 macOS, but they could still be linked on most platforms and we don’t need a *functional* mongodb-client-encryption addon to build the executable, only one with the proper source code + types)